assets/helpers/payPalExpressCheckout/payPalExpressCheckout.js

```diff
@@ -3,13 +3,20 @@
 // ----- Imports ----- //
 
 import { payPalExpressError } from 'helpers/payPalExpressCheckout/payPalExpressCheckoutActions';
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+import type { State as PayPalExpressCheckoutState } from 'helpers/payPalExpressCheckout/payPalExpressCheckoutReducer';
 
 // ----- Setup ----- //
 
 const SETUP_PAYMENT_URL = '/paypal/setup-payment';
 const CREATE_AGREEMENT_URL = '/paypal/create-agreement';
 
 
+type CombinedState = {
```

Comment:
In order to be consistent, I will always put the CombinedState definition in the reducer file.